### PR TITLE
[FIX] fee level for giftcards and bpcards

### PR DESCRIPTION
**Pull request by gabrielbazan7**

### CODE REVIEW

File: src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts [hunk 1]
```diff
@@ -515,10 +515,6 @@ export class BitPayCardTopUpPage {
                 );
                 let newAmountSat = maxAmountSat - invoiceFeeSat;
 
-                // Set expiration time for this invoice
```
Comment (cmgustavo) [inline src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts]: Why it's removed?

Comment (gabrielbazan7) [reply src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts]: That's the expiration time of the first invoice created that it's not needed

Comment (cmgustavo) [reply src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts]: Good!

**Comment by cmgustavo** [comment]
Rebase is needed

**Pull request merged**